vagrant/guest/bin/vhosts.sh

```diff
@@ -187,14 +227,22 @@ function main {
     msg "==> Generating site configuration"
     for site_path in $sites_list; do
         site_name="$(basename $site_path)"
-        
+
         site_msg=$(process_site $site_name $site_path)
         [[ -n $site_msg ]] && msg " + $site_name\n$site_msg"    # only list if process_site emitted output
     done
 
+    # TODO: For when a directory that had configs generated for it gets removed
+    # msg "==> policing old sites"
+
     if [[ ! $no_reload ]]; then
         msg -n "==> " && service httpd reload > $stdout || true    # mask the LSB exit code (expected to be 4)
         msg -n "==> " && service nginx reload > $stdout || true    # mask the LSB exit code (expected to be 4)
+        # TODO: consider trying to reload varnish to preserve cache
+        msg -n "==> " && service varnish restart > $stdout || true # mask the LSB exit code (expected to be 4)
     fi
 
+    msg "==> Building Varnish include file"
+    find /etc/varnish/sites.d/ -maxdepth 1 -name '*.vcl' -print -quit | awk '{printf "include \"%s\";\n", $1}' > /etc/varnish/includes.vcl
```

Comment:
Is it just me, or is it a mistake that this is after the varnish restart call? Updating and moving to just before the service reload block.